Makefile

```
@@ -509,16 +509,24 @@ SQLITE_BINDING_SOURCES := \
 	$(wildcard test/sqlite/*/*.c)
 
 # Implicitly depends on $(NODE_EXE), see the build-sqlite-tests rule for rationale.
+ifndef NOSQLITE
 test/sqlite/.buildstamp: $(ADDONS_PREREQS) \
 	$(SQLITE_BINDING_GYPS) $(SQLITE_BINDING_SOURCES)
 	@$(call run_build_addons,"$$PWD/test/sqlite",$@)
+else
+test/sqlite/.buildstamp:
+endif
 
+ifndef NOSQLITE
 .PHONY: build-sqlite-tests
```

Wouldn't it be PHONY in both cases?
```suggestion
.PHONY: build-sqlite-tests
ifndef NOSQLITE
```

---

@aduh95 pushed a commit to fix after I tested.